cmd/juju/commands/deploy.go

```diff
@@ -66,6 +71,14 @@ However, for local charms, when the default-series is not specified in the
 environment, one must specify the series. For example:
   local:precise/mysql
 
+For local bundles the bundle series must be always specified. For example:
```

Comment:
This is immediately contradicted by the following paragraph. How about something like:

    Local bundles can be specified either with a local:bundle/<name> URL,
    which is interpreted relative to $CHARM_DIR, or with a direct path to a
    bundle.yaml file. For example, to deploy the bundle in $CHARM_DIR/bundle/openstack,

        juju deploy local:bundle/openstack

    To deploy this using a direct path:

        juju deploy $CHARM_DIR/bundle/openstack/bundle.yaml

?